.github/workflows/aws-snp-launchmeasurement.yml

```diff
@@ -66,3 +72,21 @@ jobs:
         done
 
         jq < measurements.json
+        popd || exit 1
+
+    - name: Test equivalence sevsnpmeasure & sev-snp-measure
+      shell: bash
+      run: |
+        pushd sev-snp-measure-go/e2e || exit 1
+        go test --expected-values ../../sev-snp-measure/measurements.json --ovmf ${{ steps.build-uefi.outputs.ovmfPath }}
```

Comment:
mostly a nit, and affects the sev-snp-measure repo, not this PR:
I would prefer having this as an actual command, instead of an e2e test

Reply:
Because constellation users might want to verify? Or what motivation would there be for such a command?
Like this `sev-snp-measure compare measurements.json ovmf.fd`?

Reply:
For visibility: we decided to move test to sev-snp-measure-go repo.